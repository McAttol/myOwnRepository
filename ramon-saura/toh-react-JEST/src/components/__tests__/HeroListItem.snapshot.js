import renderer from "react-test-renderer";
import React from "react";
import HeroListItem from "../HeroListItem";
import { BrowserRouter as Router } from 'react-router-dom';


describe ('Hero List Item snapshot', () => {
    const heroListItem = renderer.create(<Router><HeroListItem/></Router>)

    it('shoul match', () => {
        expect(heroListItem.toJSON()).toMatchSnapshot();
    })
})