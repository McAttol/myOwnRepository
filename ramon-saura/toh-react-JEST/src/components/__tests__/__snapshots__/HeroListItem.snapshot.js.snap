// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero List Item snapshot shoul match 1`] = `
<p>
  HeroListItem works!
</p>
`;
